vehicle/Jeep.h

```diff
@@ -1,12 +1,13 @@
 #ifndef JEEP_H
 #define JEEP_H
 
+#include <memory>
```

Comment:
not needed here, it's already included in Car.h

Reply:
removed